lib/workers/repository/update/branch/index.ts

```diff
@@ -553,9 +553,11 @@ export async function processBranch(
     await setStability(config);
     await setConfidence(config);
 
-    // break if we pushed a new commit because status check are pretty sure pending but maybe not reported yet
+    // new commit means status check are pretty sure pending but maybe not reported yet
+    // if PR has not been created + new commit + prCreation !== immediate skip
     // but do not break when there are artifact errors
     if (
+      !branchPr &&
```

Comment:
@viceice @PhilipAbed I edited this file to restore the original ordering to make it clearer that only `!branchPr` has been added here. Unfortunately previous discussion got buried after this change

Reply:
makes sense, ill just lint fix it xD